app/models/user.rb

```diff
@@ -11,33 +11,33 @@ class User < ApplicationRecord
     allowed: "allowed",
     denied: "denied"
   }, _prefix: true
+  enum access_level: {
+    viewer: "viewer",
```

Comment:
Neat. This is where we'll also include `report_viewer` and `call_center`, right?